### PR TITLE
Made my tableView a single data source.

### PR DESCRIPTION
**Pull request by JB184351**

### CODE REVIEW

File: TalkAloud/TalkAloud/Managers/AudioManager.swift [hunk 1]
```diff
@@ -18,11 +18,19 @@ class AudioManager {
...
     private var allUniqueTags: [String] = []
-
+    
     private var didNewRecording = false
     
     private init() {}
     
+    // MARK: - Public Methods
+    
+    func loadAudioRecordings(with tags: [String]?) -> [AudioRecording] {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Managers/AudioManager.swift]: Make this optional because it will crash if you don't have any elements

**Review by virgil-adumitroae — CHANGES_REQUESTED** [review]

File: TalkAloud/TalkAloud/Managers/AudioManager.swift (hunk 1 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/Managers/AudioManager.swift]: Should I also make the audioRecrodings property in my AudioRecordingsViewController an optional so I can have a nil value anytime there are no recordings in the app?

Comment (virgil-adumitroae) [reply TalkAloud/TalkAloud/Managers/AudioManager.swift]: No. You can keep that array as empty if you don't have smth.

Comment (JB184351) [reply TalkAloud/TalkAloud/Managers/AudioManager.swift]: Done.

**Pull request merged**